web/src/components/data-test.ts

```diff
@@ -0,0 +1,193 @@
+export const AlertsPage = {
```

Comment:
To avoid aliasing imports like `import { AlertsPage as test} ...` we can use a more appropriate name:

```suggestion
export const AlertsPageTestIds = {
```